modules/adblocker/sources/statistics.es

```diff
@@ -138,30 +141,51 @@ export default class AdbStats {
   }
 
   init() {
+    browser.webNavigation.onBeforeNavigate.addListener(this.onBeforeNavigate);
```

Comment:
I have not tested it, but I wonder if you need to bind the `this` reference? `onBeforeNavigate` uses `this`, but when it is called in that way by the callback, `this` will be in a different context.

Reply:
`onBeforeNavigate` is a class property with arrow function so automatically bound to the instance